entr.c

```diff
@@ -509,6 +510,10 @@ run_utility(char *argv[]) {
 	}
 	child_pid = pid;
 
+	if (restart_opt == 1) {
+		close(stdin_pipe[0]);
+	}
```

Comment:
Remove the braces to be consistent with other conditionals that only have one statement following

Reply:
Done